webpack/scenes/ContentViews/Details/Repositories/ContentViewRepositories.js

```diff
@@ -230,6 +273,8 @@ const ContentViewRepositories = ({ cvId, details }) => {
     </DropdownItem>,
   ];
 
+  const loadingStatus = (isLoading || status === STATUS.PENDING) ? STATUS.PENDING : status;
```

Comment:
When you clicked the Show Repositories button, it would flash momentarily with the trulyEmpty state before you saw the loading spinner. This prevents that.